test/network/2020-03-01/armnetwork/models.go

```diff
@@ -10169,8 +10169,8 @@ type StringPollerResponse struct {
 // StringResponse is the response envelope for operations that return a string type.
 type StringResponse struct {
```

Comment:
There is also a `StringArrayResponse` in this code.  Not sure if it should be tagged, can you PTAL to see if it needs addressing.

Reply:
Yes the condition for adding tags was limiting to the original scalarValueName, this has been changed now. So all array types in the response envelopes will have tags. Although I'm wondering should it only be array types of scalar types? Or any array type?